### DISCUSSION TITLE
What prevents me from collecting another Plausible user's stats?

### DISCUSSION THREAD

**Discussion by nabrown** [plausible/analytics#3128] [Anything Else]
Hi there,

Let's say I notice a website, thing.com, is using Plausible.

Since there isn't a unique ID or some proof that I own a given website, what is to stop me from just setting up thing.com in my own Plausible account? Won't that website's tracking data be visible in my account (assuming they are using 'thing.com' as their domain)? What am I missing?

**Comment by Vigasaurus** [comment]
If they are using plausible cloud, and already have the domain registered in their account, you won't be able to add a site with the same name (you'll receive an error) - thus not able to have their hits deliver to your dashboard. 

The only real way you'd be able to do what you're mentioning is if you manage to register the site on plausible before the site owners, and they use the domain as you configured on their JS snippet.

**Reply by nabrown** [reply]
Thanks for this info @Vigasaurus !
